.github/workflows/release.yml

```
@@ -7,78 +7,126 @@ on:
-  #         password: ${{ secrets.DOCKERHUB_PASSWORD }}
-
-  #     - name: Add version into docker image
-  #       run: echo ${{ github.event.release.tag_name }} > LAGO_VERSION
-
-  #     - name: Build and push Docker image
-  #       uses: docker/build-push-action@v3
-  #       with:
-  #         context: .
-  #         platforms: linux/amd64
-  #         push: true
-  #         tags: getlago/api:${{ github.event.release.tag_name }}
-  #         build-args: |
-  #           SEGMENT_WRITE_KEY=${{ secrets.SEGMENT_WRITE_KEY }}
-  #           GOCARDLESS_CLIENT_ID=${{ secrets.PRODUCTION_GOCARDLESS_CLIENT_ID }}
-  #           GOCARDLESS_CLIENT_SECRET=${{ secrets.PRODUCTION_GOCARDLESS_CLIENT_SECRET }}
-  #           LAGO_OAUTH_URL=https://proxy.getlago.com
+      - name: Export Digest
+        run: |
+          mkdir -p ./_tmp/${{ github.run_id }}/${{ github.run_attempt }}/digests
+          digest="${{ steps.build.outputs.digest }}"
+          touch "./_tmp/${{ github.run_id }}/${{ github.run_attempt }}/digests/${digest#sha256:}"
+
+      - name: Upload Digest
+        uses: actions/upload-artifact@v4
+        with:
+          name: digests-${{ env.PLATFORM_PAIR }}
+          path: ./_tmp/${{ github.run_id }}/${{ github.run_attempt }}/digests/*
+          if-no-files-found: error
+          retention-days: 1
+
+      - name: Clean up
+        if: always()
+        run: |
+          [ -e ./_tmp/${{ github.run_id }}/${{ github.run_attempt }}/digests ] && \
+          rm -rf ./_tmp/${{ github.run_id }}/${{ github.run_attempt }}/digests
+
+  merge:
+    name: Merge Images
+    runs-on: lago-runner
+    needs: [build-images]
+    steps:
+      - name: Download Digests
+        uses: actions/download-artifact@v4
+        with:
+          path: ./_tmp/${{ github.run_id}}/${{ github.run_attempt }}/digests
+          pattern: digests-*
+          merge-multiple: true
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: ${{ env.REGISTRY_IMAGE }}
+          tags: |
+            type=raw,value=${{ github.event_name == 'release' && github.event.release.tag_name || github.event.inputs.version }}
+
+      - name: Set up Docker buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+      - name: Create manifest and push
+        working-directory: ./_tmp/${{ github.run_id }}/${{ github.run_attempt}}/digests
+        run: |
+          docker buildx imagetools create $(jq -cr '.tags | map("-t " + .) | join(" ")' <<< "$DOCKER_METADATA_OUTPUT_JSON") \
+            $(printf '${{ env.REGISTRY_IMAGE }}@sha256:%s ' *)
+
+      - name: Inspect Image
+        run: |
+          docker buildx imagetools inspect ${{ env.REGISTRY_IMAGE }}:${{  steps.meta.outputs.version }}
```

Is there an extra space?

```suggestion
          docker buildx imagetools inspect ${{ env.REGISTRY_IMAGE }}:${{ steps.meta.outputs.version }}
```